backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -600,32 +653,34 @@ public static Bundle createFhirBundle(
       Date currentDate,
       GitProperties gitProperties,
       String processingId) {
+
     var patientFullUrl = ResourceType.Patient + "/" + patient.getId();
-    var organizationFullUrl = ResourceType.Organization + "/" + organization.getId();
+    var testingLabOrganizationFullUrl = ResourceType.Organization + "/" + testingLab.getId();
     var practitionerFullUrl = ResourceType.Practitioner + "/" + practitioner.getId();
     var specimenFullUrl = ResourceType.Specimen + "/" + specimen.getId();
     var serviceRequestFullUrl = ResourceType.ServiceRequest + "/" + serviceRequest.getId();
     var diagnosticReportFullUrl = ResourceType.DiagnosticReport + "/" + diagnosticReport.getId();
     var deviceFullUrl = ResourceType.Device + "/" + device.getId();
 
-    var practitionerRole = createPractitionerRole(organizationFullUrl, practitionerFullUrl);
-    var provenance = createProvenance(organizationFullUrl, dateTested);
+    var practitionerRole =
+        createPractitionerRole(testingLabOrganizationFullUrl, practitionerFullUrl);
+    var provenance = createProvenance(testingLabOrganizationFullUrl, dateTested);
     var provenanceFullUrl = ResourceType.Provenance + "/" + provenance.getId();
     var messageHeader =
         createMessageHeader(
-            organizationFullUrl,
+            testingLabOrganizationFullUrl,
             diagnosticReportFullUrl,
             provenanceFullUrl,
             gitProperties,
             processingId);
     var practitionerRoleFullUrl = ResourceType.PractitionerRole + "/" + practitionerRole.getId();
     var messageHeaderFullUrl = ResourceType.MessageHeader + "/" + messageHeader.getId();
 
-    patient.setManagingOrganization(new Reference(organizationFullUrl));
+    patient.setManagingOrganization(new Reference(testingLabOrganizationFullUrl));
     specimen.setSubject(new Reference(patientFullUrl));
 
     serviceRequest.setSubject(new Reference(patientFullUrl));
-    serviceRequest.addPerformer(new Reference(organizationFullUrl));
+    serviceRequest.addPerformer(new Reference(testingLabOrganizationFullUrl));
     serviceRequest.setRequester(new Reference(practitionerRoleFullUrl));
```

Comment:
Now that this field is the practitioner instead of the org - I'm not sure where the ordering facility information will need to go.  Will probably have to wait for Carlos to get back to know for sure.  

@BobanL do you have any idea by chance?

Reply:
interesting...